### PR TITLE
[cases] AddCase hi-fi

### PR DESCRIPTION
**Pull request by ronniebeggs**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Created a `CaseSummaryContent` component from the `CaseSummaryScreen`.
- Re-used this component for the case preview on the `AddCase` screen.
- Standardized types across all related case summary components.

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- Scan a QR code to add a new case.
- Verify that the case preview matches figma designs.
- Check that the `CaseSummaryScreen` also appears as expected.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- Integrate the QR code toasts PR into the case adding flow.

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[QR Code Toasts](https://github.com/calblueprint/impact-fund/pull/52)

CC: @ronniebeggs

### CODE REVIEW

File: src/app/(BottomTabNavigation)/AllCases/CaseSummaryScreen/[caseUid].tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import { View, Text, ScrollView } from 'react-native';
+import { View, Text } from 'react-native';
 
 import styles from './styles';
+import CaseSummaryContent from '../../../../Components/CaseSummaryContent/CaseSummaryContent';
 import ExternalSiteLink from '../../../../Components/ExternalSiteLink/ExternalSiteLink';
 import { getCaseById } from '../../../../supabase/queries/cases';
 import { Case } from '../../../../types/types';
-import { formatDate } from '../utils';
+// import { formatDate } from '../utils';
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/AllCases/CaseSummaryScreen/[caseUid].tsx]: Remove unused import/comment

**Review by stephaniewong2 — COMMENTED** [review]
Looks good to me! So cleannn
